modules/statics/src/coins.ts

```diff
@@ -1117,4 +1121,44 @@ export const coins = CoinMap.fromCoins([
   teosToken('teos:IQ', 'Everipedia', 3, 'testtoken112', UnderlyingAsset.IQ, AccountCoin.DEFAULT_FEATURES, '', 'IQ'),
   teosToken('teos:BOX', 'Box', 6, 'kvszn1xyz1bu', UnderlyingAsset.EOS_BOX, AccountCoin.DEFAULT_FEATURES, '', 'BOX'),
   teosToken('teos:USDT', 'Tether', 4, 'lionteste212', UnderlyingAsset.USDT, AccountCoin.DEFAULT_FEATURES, '', 'USDT'),
+  avaxErc20('QI', 'BenQi', 18, '0x8729438eb15e2c8b576fcc6aecda6a148776c0f5', UnderlyingAsset.QI),
+  avaxErc20('XAVA', 'Avalaunch', 18, '0xd1c3f94de7e5b45fa4edbba472491a9f4b166fc4', UnderlyingAsset.XAVA),
+  avaxErc20('KLO', 'Kalao', 18, '0xb27c8941a7df8958a1778c0259f76d1f8b711c35', UnderlyingAsset.KLO),
+  avaxErc20('JOE', 'Trader Joe', 18, '0x6e84a6216ea6dacc71ee8e6b0a5b7322eebc0fdd', UnderlyingAsset.JOE),
+  arc20Token(
+    'PNG',
+    'Pangolin',
+    18,
+    '0x60781c2586d68229fde47564546784ab3faca982',
+    '2ugMhQ5ygoZ721Hjka413YkzNVGMjKQzmaDkxpeimKYEnAaWiD',
+    UnderlyingAsset.PNG
+  ),
+  arc20Token(
+    'USDT',
+    'Tether',
+    6,
+    '0xc7198437980c041c805a1edcba50c1ce5db95118',
+    '2CWanxxHzCuagcyNCqpvQP4GWrxsikGua8hv6r3dYML5DegXsm',
+    UnderlyingAsset.USDT
+  ),
+  tavaxErc20('tQI', 'BenQi', 18, '0x8729438eb15e2c8b576fcc6aecda6a148776c0f5', UnderlyingAsset.QI),
```

Comment:
In the description of tokens for testnet is usual the adding of "Test" in the fullName to distinguish them from main (production) ones:
` terc20('tcat', 'Test CAT-20 Token', 18, '0x63137319f3a14a985eb31547370e0e3bd39b03b8', UnderlyingAsset.CAT)`

Reply:
Done